tests/applications/flag_control_mixin_test.py

```diff
@@ -0,0 +1,114 @@
+from beamlime.applications.mixins import FlagControlMixin, LogMixin
+
+
+class DummyApp(LogMixin, FlagControlMixin):
+    ...
+
+
+def test_start():
+    app = DummyApp()
+    assert not app._started and app._paused
+    app.start()
+    assert app._started and not app._paused
+
+
+def test_start_twice():
+    app = DummyApp()
+    app.start()
+    assert app._started and not app._paused
+    # app.start()
+    # assert app._started and not app._paused
+
+
+def test_start_twice_paused():
+    app = DummyApp()
+    app.start()
+    app.pause()
+    app.start()
+    assert app._started and app._paused
+
+
+def test_pause():
+    app = DummyApp()
+    app.start()
+    app.pause()
+    assert app._started and app._paused
+
+
+def test_pause_twice():
+    app = DummyApp()
+    app.start()
+    app.pause()
+    app.pause()
```

Comment:
Some people might expect that calling `pause` twice pauses once and then resumes (like on e.g. a CD player).
We should make sure the behaviour of `pause` is well documented to reflect this.

Reply:
Yes, that is true.

I think the underlying interface should be as it is, but on the user-interface,
we will want the `pause` button to be grayed out or change the label as `resume` as you said.

I will put this in the `GUI` requirements.